### ISSUE TITLE
Number of columns according to updated annotation guidelines

### ISSUE THREAD

**Issue by RoxanaPatras** [distantreading/WG1#41]
About the extraction of data: 
Should all annotators have the same number of columns? 

If we add placeII syntax & role + everything around the second person (status, attribute, etc.) + everything else that has been decided to be added, then how many columns will result in the end?

**Comment by CvetanaJK** [comment]
I think that we should have the same number (and names) of columns in our tables.

**Comment by galleron** [comment]
I believe it is complicated that we decide on the number of columns now. My proposal is that everybody adds as many columns as they need, and I'll pay attention to the number of columns in the final, merged, document.

**Comment by RoxanaPatras** [comment]
Ok, thanks! 

Hope everybody is able to pre-set the value of duplicated columns. If not, it can be done from the "data validation" button. 😉

**Comment by RosarioAriasUMA** [comment]
Yes! I have been able to pre-set the value of duplicated columns.

**Comment by RoxanaPatras** [comment]
Hi, everybody! According to what has been discussed, I added 18 more columns : 4 (placeII) + 6 (personII) + 4 (otherII) + 2 (genreindicator role & syntax) + 2 (placeII rile & syntax).

I'm trying to finish tomorrow morning. I know the deadline was May, 31st but adding so many columns meant to do the annotation a 2nd time.

**Comment by CvetanaJK** [comment]
Hi, I will add the same columns as Roxana (maybe I will not need them all), in order to make thing compatible.
I will not be able to finish today . I need a few more days in order to fill new columns and I have to check the titles of all first editions - some later editions deleted the subtitles.

**Comment by DmytroYesUA** [comment]
I’ll probably add a few too, but not so many

**Comment by jberenike** [comment]
I've added the cols too

**Comment by RoxanaPatras** [comment]
> for documentation:
> 
> Conrad, der Leutnant. Eine Darstellung.
> my solution
> Conrad (person= yes, status=no, entity= Conrad, attribute=yes, role = existence, syntax= head)
> Leutnant (personII=yes, statusII=Leutnant, attributeII= no, roleII= attribution, syntaxII = apposition)
> (deu, row 67)

Hi, Berenike! Thanks for the tips! 
For the same case with apposition - say "Radu Anghel, Capitan de Talhari"/ Radu Anghel, the Captain of Thieves - I did not annotate "capitan" as a different person from "Radu Anghel"; they're one and the same entity. I mean I interpreted apposition as an explanatory sequence for the proper name. However, I did annotate as a second person "talhari" (thieves). 

Maybe we should start a new ticket for this specific situation?
